spec/models/host_factory_spec.rb

```diff
@@ -88,5 +88,54 @@
         expect(host.role.memberships_as_member.map(&:role)).to eq(host_factory.role.layers)
       end
     end
+
+    describe 'verify create host given AUTHN_API_KEY config' do
+      context 'when CONJUR_AUTHN_API_KEY_DEFAULT is true' do
+        before do
+          allow(ENV).to receive(:[]).with('CONJUR_AUTHN_API_KEY_DEFAULT').and_return('true')
+          allow(Rails.application.config.conjur_config).to receive(:authn_api_key_default).and_return(true)
```

Comment:
I agree with @doodlesbykumbi  on doubling the config object itself. With this I don't believe you need the line above to double ENV, do you? (Same on line 116)

Reply:
done